nova_plugin/server.py

```diff
@@ -222,6 +227,11 @@ def rename(name):
             # will fail.
             server['nics'].remove({'net-id': management_network_id})
 
+            # make management network the first interface
+            idx = port_network_ids.index(management_network_id)
+            server['nics'] = server.get('nics', [])
+            server['nics'].insert(0, {'port-id': port_ids[idx]})
+            nics.pop(idx)
```

Comment:
doesn't this line actually pop out the wrong network?
if im not mistaken this should be (idx+1), since idx has been pushed one due to the insert at 0